mlflow/dspy/autolog.py

```diff
@@ -20,6 +20,7 @@ def autolog(
     log_evals: bool = False,
     disable: bool = False,
     silent: bool = False,
+    auto_logged_model_creation: bool = True,
```

Comment:
```suggestion
    create_logged_model: bool = True,
```

Can we rename this argument? `auto_logged_model_creation` is a bit long.